open-budget/app/styles/main.less

```diff
@@ -4,28 +4,39 @@
 //@import url("http://fonts.googleapis.com/earlyaccess/alefhebrew.css");
```

Comment:
Did you (by mistake) auto-format this entire file?

It makes it impossible to review (+some of the auto-formattings reduce the readability)

Please revert this...